src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.html

```diff
@@ -1 +1,2 @@
-{{ action.name }}
+<div #title class="pt-1 ps-2 pe-2">{{ action.name }}</div>
+<img *ngIf="isVideoClipAction() && videoClipThumbnailUrl" [ngSrc]="videoClipThumbnailUrl" [width]="getThumbnailWidth()" [height]="getThumbnailHeight()" [alt]="action.name">
```

Comment:
Couldn't the condition be reduced to `videoClipThumbnailUrl` since it is only set, when it is a video clip action?

Reply:
Yes, it could.
I do however like the explicitness that comes with `isVideoClipAction()` that easily shows that it most be a VideoClipAction. Only using `videoClipThumbnail` would make it implicit that it must be a VideoClipAction.

Reply:
Personally, I think that the explicitness here can lead to confusion about when the `videoClipThumbnailUrl` is available and adds unnecessary tokens that a reader needs to mentally digest.